aws/resource_aws_macie2_invitation_accepter_test.go

```diff
@@ -0,0 +1,205 @@
+			return fmt.Errorf("no administrator account found for: %s", resourceName)
+		}
+
+		return nil
+	}
+}
+
+func testAccCheckAwsMacie2InvitationAccepterDestroy(s *terraform.State) error {
+	conn := testAccProvider.Meta().(*AWSClient).macie2conn
+
+	for _, rs := range s.RootModule().Resources {
+		if rs.Type != "aws_macie2_invitation_accepter" {
+			continue
+		}
+
+		input := &macie2.GetAdministratorAccountInput{}
+		output, err := conn.GetAdministratorAccount(input)
+		if tfawserr.ErrCodeEquals(err, macie2.ErrCodeResourceNotFoundException) ||
+			tfawserr.ErrMessageContains(err, macie2.ErrCodeAccessDeniedException, "Macie is not enabled") {
+			continue
+		}
+
+		if output == nil || output.Administrator == nil || aws.StringValue(output.Administrator.AccountId) != rs.Primary.Attributes["administrator_account_id"] {
+			continue
+		}
+
+		return fmt.Errorf("macie InvitationAccepter %q still exists", rs.Primary.ID)
+
+	}
+
+	return nil
+
+}
+
+func testAccAwsMacieInvitationAccepterConfigBasic(email string) string {
+	return testAccAlternateAccountProviderConfig() + fmt.Sprintf(`
+data "aws_caller_identity" "primary" {
+  provider = "awsalternate"
+}
+
+data "aws_caller_identity" "current" {}
+
+resource "aws_macie2_account" "primary" {
+  provider = "awsalternate"
+}
+
+resource "aws_macie2_account" "member" {}
+
+resource "aws_macie2_member" "primary" {
+  provider   = "awsalternate"
+  account_id = data.aws_caller_identity.current.account_id
+  email      = %[1]q
+  depends_on = [aws_macie2_account.primary]
+}
+
+resource "aws_macie2_invitation" "primary" {
+  provider   = "awsalternate"
+  account_id = data.aws_caller_identity.current.account_id
+  depends_on = [aws_macie2_member.primary]
+}
+
+resource "aws_macie2_invitation_accepter" "test" {
+  administrator_account_id = data.aws_caller_identity.primary.account_id
+  depends_on               = [aws_macie2_invitation.primary]
+}
+
+`, email)
+}
+
+func testAccAwsMacieInvitationAccepterConfigMemberStatus(email, memberStatus string) string {
+	return testAccAlternateAccountProviderConfig() + fmt.Sprintf(`
+data "aws_caller_identity" "primary" {
+  provider = "awsalternate"
+}
+
+data "aws_caller_identity" "current" {}
+
+resource "aws_macie2_account" "primary" {
+  provider = "awsalternate"
+}
+
+resource "aws_macie2_account" "member" {}
+
+resource "aws_macie2_member" "primary" {
+  provider   = "awsalternate"
+  account_id = data.aws_caller_identity.current.account_id
+  email      = %[1]q
+  status     = %[2]q
+  depends_on = [aws_macie2_account.primary]
+}
+
+resource "aws_macie2_invitation" "primary" {
+  provider   = "awsalternate"
+  account_id = data.aws_caller_identity.current.account_id
+  depends_on = [aws_macie2_member.primary]
+}
+
+resource "aws_macie2_invitation_accepter" "test" {
+  administrator_account_id = data.aws_caller_identity.primary.account_id
+  depends_on               = [aws_macie2_invitation.primary]
+}
```

Comment:
Unfortunately, this doesn't work because a number of the acceptance testing framework functions only work on the primary account. Depending on the resource, this will be different.